src/Sulu/Bundle/AdminBundle/Resources/js/components/Form/tests/Form.test.js

```diff
@@ -4,5 +4,10 @@ import {render} from 'enzyme';
 import Form from '../Form';
 
 test('Render a form', () => {
-    expect(render(<Form>Test</Form>)).toMatchSnapshot();
```

Comment:
Actually I am always trying to keep the number snapshot tests minimal, because they cause a lot of noise when something fails... This is about rendering the component without a class, I think flow got already covered most of that by making sure that something is set resp. that it works without a `skin` being set.

Reply:
okay, dont hae a strong opinion on that 🙂